src/examples/generation.ts

```diff
@@ -15,19 +16,19 @@ const renderer: Renderer = new Renderer({
     width: 800,
     height: 600,
     maxLights: 2,
-    ambientLightColor: RGBColor.fromRGB(51, 51, 51)
+    ambientLightColor: RGBColor.fromRGB(90, 90, 90)
 });
 
 // Create light sources for the renderer
 const light1: Light = new Light({
     position: { x: 10, y: 10, z: 10 },
     color: RGBColor.fromHex('#FFFFFF'),
-    intensity: 256
+    strength: 100
```

Comment:
Is there a reason for using `strength` over `intensity`?

Reply:
Intensity as it was used before was like, how focused the light is, whereas this is just increasing overall power.